tests/gocase/unit/type/zset/zset_test.go

```diff
@@ -439,6 +440,23 @@ func basicTests(t *testing.T, rdb *redis.Client, ctx context.Context, encoding s
 			{4, "d"},
 		}, rdb.ZRangeWithScores(ctx, "ztmp", 0, -1).Val())
 
+		for i := 0; i < 10; i++ {
+			cmd := rdb.ZRangeArgs(ctx, redis.ZRangeArgs{Key: "ztmp", Count: 0, ByScore: true, Start: 0, Stop: -1, Offset: int64(i)})
+			require.Equal(t, []string{}, cmd.Val())
```

Comment:
https://github.com/redis/go-redis/blame/0bdc7dd8986e4923b3309f880245b9311d09f273/commands.go#L2790 won't append the `LIMIT offset,count` if the offset and counter are zero. So for the kvrocks side, it will regard it as the default value which is offset = 0 and count = -1.

By the way, we should check the error first before comparing the value.

Reply:
Thanks. I didn't read the process in go-redis.